<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="m_mainWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="3">
     <widget class="QWidget" name="m_optionsWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="m_doubleOffsetLabel">
         <property name="text">
          <string>Double-stranded offset:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="m_singleOffsetLabel">
         <property name="text">
          <string>Single-stranded offset:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_shapeFileLabel">
         <property name="text">
          <string>SHAPE restraints (.shape):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="m_doubleOffsetSelected"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_diferentialShapeFileLabel">
         <property name="text">
          <string>Differential SHAPE:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="m_singleOffsetSelected"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="m_differencialShapeFileSelected"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="m_shapeFileSelected"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="m_shapeBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="m_diferrentialShapeBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="m_doubleOffsetBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="m_singleOffsetBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QWidget" name="m_spinBoxedWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="3">
           <widget class="QLabel" name="m_siLabel">
            <property name="text">
             <string>SI:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4">
           <widget class="QDoubleSpinBox" name="m_smDoubleSpinBox">
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>1.800000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QDoubleSpinBox" name="m_siDoubleSpinBox">
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>-0.600000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="m_mLabel">
            <property name="text">
             <string>M:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="m_smLabel">
            <property name="text">
             <string>SM:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="m_dsmDoubleSpinBox">
            <property name="value">
             <double>2.110000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="m_imLabel">
            <property name="text">
             <string>IM:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="m_iwSpinBox">
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_mspinBox">
            <property name="maximum">
             <number>1000000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="m_dhmLabel">
            <property name="text">
             <string>DSM:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="m_wLabel">
            <property name="text">
             <string>W:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="m_ipSpinBox">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="m_imSpinBox">
            <property name="maximum">
             <number>100000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="m_ipLabel">
            <property name="text">
             <string>IP:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="m_iwLabel">
            <property name="text">
             <string>IW:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QSpinBox" name="m_wSpinBox"/>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="m_p2Label">
            <property name="text">
             <string>P2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QDoubleSpinBox" name="m_p2DoubleSpinBox">
            <property name="value">
             <double>0.650000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QDoubleSpinBox" name="m_p1DoubleSpinBox">
            <property name="value">
             <double>0.350000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="m_p1Label">
            <property name="text">
             <string>P1:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="m_pLabel">
            <property name="text">
             <string>P:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QSpinBox" name="m_pSpinBox">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="m_phLabel">
            <property name="text">
             <string>PH:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="6">
           <widget class="QSpinBox" name="m_phSpinBox">
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="m_runButton">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QTextEdit" name="m_commandOutput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="m_mainOptionsWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_inputSelected"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_inputLabel">
         <property name="text">
          <string>Input file (.seq):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_inputBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_outputLabel">
         <property name="text">
          <string>Output file (.ct):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_outputSelected"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="m_outputBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="m_optionsCheckBox">
      <property name="text">
       <string>Optional parameters</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="m_actionAbouts"/>
   </widget>
   <widget class="QMenu" name="m_menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="m_actionSettings"/>
    <addaction name="m_actionDraw"/>
   </widget>
   <addaction name="m_menuFile"/>
   <addaction name="m_menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="m_mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
  <action name="m_actionAbouts">
   <property name="text">
    <string>About ShapeKnots-GUI</string>
   </property>
  </action>
  <action name="m_actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="m_actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="m_actionDraw">
   <property name="text">
    <string>Draw</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
